### ISSUE TITLE
Add interactions in case of missing chat (if not used on Youtube/Twitch...)

### ISSUE THREAD

**Issue by asaurat** [uurshin/minotaure#63]
When used outside of Twitch (for example during an IRL conference), some people will talk, others won't. For people who don't want to talk, it'd be great to allow for other means of communicating.

The main objective could be to add an integrated chat (optional as it would be useless with Twitch/YT).

Secondary ways to add this interaction : allowing for simple emoji reactions, allowing PC dice rolls (this could be complex and need a specific ticket).
